src/components/List/List.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Card from "../Card/Card";
+import "./List.css";
+
+const List = ({ quotes, key }) => {
+  console.log(quotes, "quotes");
+  let quoteCards = quotes.map((quote) => {
+    console.log(quote);
+    return <Card dialog={quote.dialog} key={key} />;
```

Comment:
You have an id in the Card component, could you use that as the value for key here?

Reply:
If i take that out I still get this error: Warning: List: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)